core/theme/dark.js

```diff
@@ -131,10 +131,32 @@ Blockly.Themes.Dark.setComponentStyle('scrollbarOpacity', 0.4);
 /**
  * CSS for the dark theme.
```

Comment:
Since this is our main example currently for how to use themes could we update the comment to explain a bit more about what this function is doing?

Reply:
Done. Good idea!